pkg/models/user_token.go

```diff
@@ -33,7 +34,7 @@ type RevokeAuthTokenCmd struct {
 
 // UserTokenService are used for generating and validating user tokens
 type UserTokenService interface {
-	CreateToken(ctx context.Context, userId int64, clientIP net.IP, userAgent string) (*UserToken, error)
```

Comment:
What prompted this change? It doesn't make sense to me as we're now mixing db calls and request logic in the same method, making the scope of this method really large.